keps/sig-cli/5295-kyaml/README.md

```diff
@@ -1188,6 +1189,9 @@ to KYAML, parses that KYAML back into the original type (with the standard YAML
 decoder), and compares the original and result objects. Doing this enough times
 should give us fairly high confidence.
 
+- `sigs.k8s.io/yaml`: `2025-07-24` - `99.9%`
+- `k8s.io/cli-runtime/pkg/printers/`: `2025-07-25` - `0.1%`
```

Comment:
Looking at https://testgrid.k8s.io/sig-testing-canaries#ci-kubernetes-coverage-unit I'm seeing 66.4%, not what you have here.